src/absil/il.fs

```diff
@@ -77,21 +81,21 @@ let rec splitNamespaceAux (nm:string) =
 
 /// Global State. All namespace splits ever seen
 // ++GLOBAL MUTABLE STATE
+#if FABLE_COMPILER
```

Comment:
No reason, just looked cleaner this way. This too can go away after Dictionary constructors can be handled properly by Fable.